docs: add detach subscription link to README

yoshi-automation opened the pull request:
This PR was generated using Autosynth. :rainbow:

Synth log will be available here:
https://source.cloud.google.com/results/invocations/15e3b9fd-f00a-4ff5-9786-abfb8d921425/targets

- [ ] To automatically regenerate this PR, check this box.

feywind commented:
We're still waiting on the actual feature release, so I don't want to make this quite this visible yet. Let's hold off on this if we can.

yoshi-automation committed:
changes without context

        autosynth cannot find the source of changes triggered by earlier changes in this
        repository, or by version upgrades to tools such as linters.
